ci.mk

```diff
@@ -1,6 +1,11 @@
 # SPDX-License-Identifier: Apache-2.0
```

Comment:
This is not working when invoking the ci on the bao-libvmm repo using docker.

The output is the following:
```
$ make -C ci/docker misra-check PLATFORM=qemu-aarch64-virt
make: Entering directory '/home/david/Projects/bao/bao-vmm-static/bao-libvmm/ci/docker'
Launching docker container...
latest: Pulling from baoproject/bao
Digest: sha256:105ffac642d2c5e6de658e9e043cefd4b3755c09530772a2e4a83aac42cb8d15
Status: Image is up to date for baoproject/bao:latest
ci/ci.mk:4: util.mk: No such file or directory
make: *** No rule to make target 'util.mk'.  Stop.
make: [Makefile:27: misra-check] Error 2 (ignored)
Leaving and destroying docker container...
make: Leaving directory '/home/david/Projects/bao/bao-vmm-static/bao-libvmm/ci/docker'

```

It is also not executing correctly locally. Here is the output:

```
$ make -C ci misra-check PLATFORM=qemu-aarch64-virt
make: Entering directory '/home/david/Projects/bao/bao-vmm-static/bao-libvmm/ci'
cppcheck: error: no C or C++ source files found.
make: *** [ci.mk:238: misra-check] Error 1
make: Leaving directory '/home/david/Projects/bao/bao-vmm-static/bao-libvmm/ci'
```

Reply:
Included the util.mk file using an absolute path. Should be working

Reply:
It is working for me